apis/Google.Cloud.Diagnostics.AspNetCore/docs/index.md

```diff
@@ -246,3 +246,34 @@ that is propagated in a `custom_trace_id` header. This is of no use in the real
 
 {{sample:Trace.CustomTraceContext}}
 
+## Using Google Trace in other than ASP.NET Core applications
```

Comment:
"in other than ASP.NET Core applications" => "in applications not based on ASP.NET Core" perhaps? Not sure.